### ISSUE TITLE
=

### ISSUE THREAD

**Issue by xqx456** [zou-yawen/Dataset-Distillation-via-Vision-Language-Category-Prototype#1]
Thank you for sharing this amazing work on VLCP. I've gone through the paper/repository and I'm very impressed by the results.

To better understand and experiment with your method, I was wondering if it would be possible for you to release the train_text_to_image.py or the pre-trained diffusion model.

**Comment by zou-yawen** [comment]
Here are the download links for our resources:

https://gofile.me/6UrIQ/zaQO2ICY5

Trained models — Stable Diffusion fine-tuned with LLaVA-generated semantics (via diffusers 0.30.0.dev0).

Distilled datasets — ImageNet-1K and subsets distilled with the above models.

Each model corresponds to a fixed random seed {0, 1, 2} to support reproducible sampling and evaluation.

**Comment by xqx456** [comment]
Thank you very much for your reply. I have solved the problem. I believe VLCP will expand its influence and look forward to your next work.
